### PR TITLE
Free Worlds Typo Corrections

### PR DESCRIPTION
**Pull request by McloughlinGuy**
NOTICE: Delete the sections that do not apply to your PR, and fill out the section that does.
(You can open a PR to add or improve a section, if you find them lacking!)

----------------------
**Content (Artwork / Missions / Jobs)**

## Summary
I fixed a very large amount of typos in the free worlds storyline. The whole process of finding all of these took me a bout an hour. Notes: In system was changed to in the system in many cases, but it could be changed back if deemed necessary. I ran all these missions through the google docs spelling check, however I did not change them all according to that, rather did a deeper check of the things it said were issues.

### CODE REVIEW

File: data/human/free worlds intro.txt [hunk 1]
```diff
@@ -1327,7 +1327,7 @@ mission "FW Escort 1"
 				accept
 	
 	on accept
-		log "Serving as an escort for a Free Worlds supply convoy to Tarazed. It sounds like pirates have been specifically targeting their convoys, recently."
+		log "Serving as an escort for a Free Worlds supply convoy to Tarazed. It sounds like pirates have been blatantly targeting their convoys recently."
```
Comment (Arrow2thekn33) [inline data/human/free worlds intro.txt:1330]: I agree that this needs to be changed, but I'm not sure blatantly captures the right meaning. You could just remove the word altogether, or find one that fits better.

**Review by Arrow2thekn33 — APPROVED** [review]
Good changes. I've also been working on a typo fix PR, and I ran into a similar issue with "in system" vs "in the system". There are many more instances of both phrases, so we should probably pick one to standardize on.

File: data/human/free worlds intro.txt (hunk 1 above)
Comment (Zitchas) [reply data/human/free worlds intro.txt:1330]: I'm not sure this is needed. The Pirates could very well be specifically targeting them.

File: data/human/free worlds checkmate.txt [hunk 2]
```diff
@@ -931,7 +931,7 @@ mission "FWC Checkmate 1C"
 				`	"Understood. We will depart immediately."`
 					accept
 				`	"Are we really sure we want to attack Earth?"`
-			`	"Have they given us any other choice?" asks Tomek. "We tried diplomacy. And we cannot hope to defeat the entire Navy. So, unless we wish to surrender and all be put on trial as war criminals, we need to press on and gain the upper hand in the only way we can. The Senate and the Council are agreed on this." You have no choice but to follow his orders and to begin getting your fleet ready for the next attack.`
+			`	"Have they given us any other choice?" asks Tomek. "We tried diplomacy. And we cannot hope to defeat the entire Navy. So, unless we wish to surrender and all be put on trial as war criminals, we need to press on and gain the upper hand in the only way we can. The Senate and the Council are in agreement on this." You have no choice but to follow his orders and to begin getting your fleet ready for the next attack.`
```
Comment (Zitchas) [inline data/human/free worlds checkmate.txt:934]: I don't know if this is *good* English, but it is definitely reasonable conversational dialog to say that x and y are agreed on something.

File: data/human/free worlds intro.txt (hunk 1 above)
Comment (McloughlinGuy) [reply data/human/free worlds intro.txt:1330]: > I'm not sure this is needed. The Pirates could very well be specifically targeting them.

They are specifically targeting them, except blatantly seems like a better choice of word than specifically.

File: data/human/free worlds checkmate.txt [hunk 3]
```diff
@@ -1027,9 +1027,9 @@ mission "FWC Pug 1"
 				goto end
 			
 			label reinforcements
-			`	"Yes," he says, "and a second after they arrived in system, this other fleet of ships appeared, unlike any ships I've ever seen before. They're tearing the Wolf Pack fleet to ribbons. If you don't get up there immediately, none of them will be left alive."`
+			`	"Yes," he says, "and a second after they arrived in the system, this other fleet of ships appeared, unlike any ships I've ever seen before. They're tearing the Wolf Pack fleet to ribbons. If you don't get up there immediately, none of them will be left alive."`
```
Comment (tehhowch) [inline data/human/free worlds checkmate.txt:1030]: "in-system," or could drop the prepositional phrase entirely

File: data/human/free worlds epilogue.txt [hunk 4]
```diff
@@ -63,7 +63,7 @@ mission "FW Epilogue: Alondo"
 	
 	on offer
 		conversation
-			`As you're coming in for a landing on Bourne, you receive a message from Alondo. "<first>!" he says. "I just heard that you were in system. Want to meet up for a drink and talk about old times?"`
+			`As you're coming in for a landing on Bourne, you receive a message from Alondo. "<first>!" he says. "I just heard that you were in the system. Want to meet up for a drink and talk about old times?"`
```
Comment (tehhowch) [inline data/human/free worlds epilogue.txt:66]: Unnecessary to change this one

File: data/human/free worlds intro.txt [hunk 5]
```diff
@@ -858,7 +858,7 @@ mission "FW Katya 3"
...
+			`	"Well," she says, "it started when the Syndicate began experimenting with fully automated mining facilities on uninhabitable planets. Pirates started raiding the mines to steal equipment, and the Syndicate claimed that these installations were 'colonies' and it was the Navy's job to defend them. The Act was Parliament's response, stating that taking goods from a human being is piracy, but if the owner or some living representative of theirs is not present in the system, taking those goods is a legal salvage operation. The point was to keep the Navy from being obligated to defend every equipment cache or abandoned station in the galaxy. But by the same law, those Navy surveillance drones become abandoned goods the moment their motherships leave the system."`
```
Comment (tehhowch) [inline data/human/free worlds intro.txt:861]: This one is fine.

File: data/human/free worlds middle.txt [hunk 6]
```diff
@@ -392,7 +392,7 @@ event "battle for Rastaban"
 
 mission "FW Southern Battle 2"
 	name "Attack Rastaban"
-	description "Travel directly to the <system> system, so that you will arrive on <day> and join the battle against the Navy. Disable or destroy all Navy ships in system."
+	description "Travel directly to the <system> system, so that you will arrive on <day> and join the battle against the Navy. Disable or destroy all Navy ships in the system."
```
Comment (tehhowch) [inline data/human/free worlds middle.txt:395]: Also fine.

**Review by tehhowch — COMMENTED** [review]
IMO "in-system" is more fitting than "in the system" when describing the location of ships (compare with "out-of-system").

It's also fair to believe that a galaxy-spanning civilization would drop the definite article, especially in speech, making "in system" just as correct as "in-system" and "in the/this system."

File: data/human/free worlds intro.txt (hunk 1 above)
Comment (TheMarksman-ES) [reply data/human/free worlds intro.txt:1330]: "Have specifically been targeting their convoy recently" would be the better phrasing here. While blatantly does have its own use case, specifically would imply something more coordinated, say, as revenge for banding together and shooting at them.

**Comment by TheMarksman-ES** [comment]
Notes: In the system, whilst grammatically correct, does not apply into certain settings. It's a pain when I've to write stories and projects and it changes or suggest stuff that are correct to "the system."

"In system" for ES is correct. And shouldn't be changed.

**Comment by McloughlinGuy** [comment]
> Notes: "In the system," whilst grammatically correct, does not apply into certain settings. It's a pain when I've to write stories and projects and it changes or suggest stuff that are correct to "the system."
> 
> "In system" for ES is correct. And shouldn't be changed.

I changed the things tehhowch said weren’t typos back to their original state.

File: data/human/free worlds intro.txt (hunk 1 above)
Comment (Amazinite) [reply data/human/free worlds intro.txt:1330]: There's a lot more intentionality behind the word "specifically" as opposed to "blatantly." "Specifically" suggests to me that Free Worlds convoys are being targeted to the exclusion of other potential targets. "Blatantly" doesn't hold that same weight.

Comment (Amazinite) [inline data/human/free worlds intro.txt:1330]:
```suggestion
		log "Serving as an escort for a Free Worlds supply convoy to Tarazed. It sounds like pirates have been specifically targeting their convoys recently."
```

**Pull request merged**